<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Validation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="name_input">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Enter your name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_email">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="email_input">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Enter your email</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_age">
        <property name="text">
         <string>Age:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="age_input">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Enter your age</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_phone">
        <property name="text">
         <string>Phone Number:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="phone_input">
    <property name="minimumSize">
        <size>
            <width>300</width>
            <height>30</height>
        </size>
    </property>
    <property name="placeholderText">
        <string>+62 999 9999 9999</string>
    </property>
    <property name="inputMask">
        <string></string>
    </property>
</widget>

      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_address">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="address_input">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>120</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Enter your address</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_gender">
        <property name="text">
         <string>Gender:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="gender_combo">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string></string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_education">
        <property name="text">
         <string>Education:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="education_combo">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string></string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High School</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diploma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bachelor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Master</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PhD</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #4CAF50; color: white; padding: 10px; border: none;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f44336; color: white; padding: 10px; border: none;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="student_info">
      <property name="text">
       <string>Created by: I Nengah Dwi Putra | Student ID: F1D022049</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 12px; color: #555;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>